README.md

```diff
@@ -1,4 +1,4 @@
-TangeloHub [![Build Status](https://travis-ci.org/Kitware/tangelohub.png?branch=master)](https://travis-ci.org/Kitware/tangelohub)
+TangeloHub [![Build Status](https://travis-ci.org/Kitware/flow.png?branch=master)](https://travis-ci.org/Kitware/flow)
```

Comment:
Fixed, added some documentation instances of tangelohub - also should tangelohub.org references be changed?

Blaming everything else on Monday.